services/blockchain-indexer/shared/utils/paramUtils.js

```diff
@@ -17,14 +17,26 @@ const {
 	Exceptions: { ValidationException },
 } = require('lisk-service-framework');
 
+const EMPTY_STRING = '';
```

Comment:
Please ensure that the changes recommended in PR #1129 ([here](https://github.com/LiskHQ/lisk-service/pull/1129/files/153b165b811457cd61db6602ba3171528517617b#r884799842)) are also reflected here.